packages/datasources/src/mainnet.ts

```diff
@@ -145,15 +146,37 @@ export default {
         startBlock: 17575699,
       },
       RegistrarController: {
+        /**
+         * This controller was removed from BaseRegistrar contract
+         * at block 35936564 with the following tx:
+         * https://basescan.org/tx/0x88a3cc03291bb1a4b2bd9ccfe6b770988470001905d96c32bd41b866797b684b
+         */
         abi: base_RegistrarController,
         address: "0x4ccb0bb02fcaba27e82a56646e81d8c5bc4119a5",
         startBlock: 18619035,
```

Comment:
should we consider adding an `endBlock: 35936564` to this contract since it was removed? i don't think any transactions would be successful but maybe worth adding? i don't think it would be, but if this causes an issue with indexing or our handling of the generated ponder config, let's not bother